src/test/java/seedu/address/ui/actions/AddStudentTest.java

```diff
@@ -0,0 +1,29 @@
+package seedu.address.ui.actions;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.Test;
+import org.testfx.api.FxAssert;
+import org.testfx.framework.junit5.ApplicationTest;
+
+import javafx.scene.Node;
+import javafx.stage.Stage;
+import seedu.address.MainApp;
+
+public class AddStudentTest extends ApplicationTest {
+    @Override
+    public void start(Stage stage) {
+        MainApp mainApp = new MainApp();
+        try {
+            mainApp.init();
+        } catch (Exception e) {
+            assertTrue(false);
+        }
+        mainApp.start(stage);
+    }
+
+    @Test
+    public void commandTextFieldExists() {
+        FxAssert.verifyThat("#commandTextField", Node::isVisible);
+    }
```

Comment:
Here's an example on how to test if UI elements exist. We can use this to do things like simulate user input and then check if certain elements exist.